Assets/HoloToolkit/Utilities/Tests/StabilizationPlaneSetting.unity

```diff
@@ -0,0 +1,1372 @@
+%YAML 1.1
```

Comment:
@jwittner any preference if this test scene should be in Utilities\Tests vs Input\Tests?